extension-manifest-v3/src/store/metrics.js

```diff
@@ -0,0 +1,41 @@
+import { store } from 'hybrids';
+import { Metrics as Telemetry } from '@ghostery/libs';
+
+const VELOCITIES = [
+  'active_daily_velocity',
+  'engaged_daily_velocity',
+  'engaged_daily_count',
+];
+
+const Metrics = {
+  ...VELOCITIES.reduce((all, current) => ({ ...all, [current]: [0] }), {}),
+  [store.connect]: {
+    async get() {
+      const { metrics = {} } = await chrome.storage.local.get(['metrics']);
+      VELOCITIES.forEach((velocity) => {
+        if (
+          (metrics[velocity] &&
+            metrics[velocity].length === 0 &&
```

Comment:
"length !== 0"? (Next check is accessing the first element)
Or should there be "||" instead of "&&"?